lib/parameters.py

```diff
@@ -58,6 +58,78 @@ def gen_avc_cqp_parameters(spec, profiles):
   params = gen_avc_cqp_variants(spec, profiles)
   return keys, params
 
+def gen_avc_bdepth_variants(spec, profiles):
+  for case, params in spec.items():
+    variants = copy.deepcopy(params.get("variants", dict()).get("cqp", None))
```

Comment:
We shouldn't reuse the "cqp" variant key because test configs already define cases for this. If we reuse it, then we suddenly double the number of "cqp" test cases.
Define a new key, instead (e.g. bdepth).